Products/PluggableAuthService/plugins/tests/ChallengeProtocolChooser.txt

```diff
@@ -145,9 +145,9 @@ And with valid credentials the reqeuest should succeed:
   ... </methodCall>
   ... """ % (folder_name, username, password), handle_errors=True)
   HTTP/1.1 200 OK
+  ...
   Content-Length: 140
   Content-Type: text/xml
-  <BLANKLINE>
```

Comment:
This doesn't look correct:  there *should* be a blank line between the headers and the payload.